Add procedure, concept, and reference for image mode hosts (#3663)
theforeman/foreman-documentation: guides/common/modules/ref_managing-image-modes-hosts.adoc

[id="Managing_image_mode_hosts{context}"]

> = Managing image mode hosts 

Reviewer: ```suggestion
= Remote execution jobs for image mode hosts 
```
Gerundium is for procedures.

> Use the new remote execution jobs under the Bootc job category to manage image mode hosts:

Reviewer: ```suggestion
You can use remote execution jobs under the *Bootc* job category to manage image mode hosts:
```

** *Bootc Action* - Script Default: Select any Bootc action from the dropdown menu, enter arguments, and specify a target.
** *Bootc Rollback* - Script Default: Run Bootc rollback to revert to a previous image.
** *Bootc Status* - Script Default: Run Bootc status with optional extra parameters to check the status.
** *Bootc Switch* - Script Default: Run Bootc switch with a mandatory image input and optional extra parameters to switch to a different image.

> ** *Bootc Upgrade* - Script Default: Run Bootc upgrade with optional extra parameters to update the image.

Reviewer: Can we cross-link this with REX for more info?

```suggestion
** *Bootc Upgrade* - Script Default: Run Bootc upgrade with optional extra parameters to update the image.

.Additional resources
* <link to REX chapter here>
```
